eisbuk-admin/src/components/atoms/IntervalCard/__tests__/IntervalCard.test.tsx

```diff
@@ -1,16 +1,75 @@
 import React from "react";
-import { render } from "@testing-library/react";
+import {cleanup, screen render } from "@testing-library/react";
 
 import IntervalCard from "../IntervalCard";
+import * as bookingActions from "@/store/actions/bookingOperations";
+import { bookedSlot, nonBookedSlot } from "../__testData__/dummyData";
 
-import { baseProps } from "../__testData__/dummyData";
+import { Customer, Slot } from "eisbuk-shared";
+import { SlotInterface, SlotInterval } from "@/types/temp";
+
+
+const mockDispatch = jest.fn();
+
+jest.mock("react-redux", () => ({
+  ...jest.requireActual("react-redux"),
+  useDispatch: () => mockDispatch,
+  /** We're mocking this not to fail tests including SlotForm (this will be returning newSlotTime as null) @TODO remove in the future */
+  useSelector: () => null,
```

Comment:
If you were using selector inside of the component, always returning `null` wold produce errors in most cases. However you're not using `useSelector` within the component, so this is unnecessary as well.